### DISCUSSION TITLE
Alist V3.7.2 挂载蓝奏云无法使用

### DISCUSSION THREAD

**Discussion by Meeeeeet6** [AlistGo/alist#2875] [Q&A]
报错如下图所示：
![image](https://user-images.githubusercontent.com/74605260/210194685-5fbda4a7-271c-4410-bf26-90ff9f191d12.png)
![image](https://user-images.githubusercontent.com/74605260/210194688-2c4e890a-00ff-491c-86dd-97ffe913a19c.png)
![image](https://user-images.githubusercontent.com/74605260/210194695-a5862c6f-d15c-4376-920c-f7094677d9be.png)

**Comment by ikun09** [comment]
我也是
